client/src/components/RelatedAndComparison/assets/RelatedProducts.jsx

```diff
@@ -1,65 +1,53 @@
 /* eslint-disable no-plusplus */
 /* eslint-disable prefer-destructuring */
 /* eslint-disable react/prop-types */
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { FaArrowAltCircleLeft, FaArrowAltCircleRight } from 'react-icons/fa';
+import Images from './Images.jsx';
 
 const RelatedProducts = ({ styles, handleClick }) => {
   const [currentCard, setCurrentCard] = useState(0);
-  const [cardDisplay, setCardDisplay] = useState([]);
   const length = styles.length;
 
-  useEffect(() => {
-    const display = [];
-
-    if (styles.length >= 1) {
-      for (let i = currentCard; i < (currentCard + 5); i++) {
-        if (currentCard === 0) {
-          display.push(styles[i]);
-        } else {
-          display.push(styles[i - 1]);
-        }
-      }
-    }
-    setCardDisplay(display);
-  }, [styles, currentCard]);
-
-  const prevSlide = () => {
+  const prevCard = () => {
     setCurrentCard(currentCard === 0 ? 0 : currentCard - 1);
   };
-  const nextSide = () => {
+  const nextCard = () => {
     setCurrentCard(currentCard === (length - 1) ? 0 : currentCard + 1);
   };
 
+  const cards = styles.map((item) => {
+    const originalPrice = item.originalPrice;
+    const salePrice = item.salePrice;
+
+    return (
+      <div key={item.styleID}>
+        <Images images={item.styleImages} />
+        <button type="button" className="relatedBtn" onClick={() => { handleClick(item.styleID); }}>&#9733;</button>
+        {/* <span className="relatedBtn">&#9733;</span> */}
+        <br />
+        <span className="item-category">{item.category}</span>
+        <br />
+        <span className="item-name">{item.name}</span>
+        <br />
+        <span className={salePrice ? 'cross-out-price' : 'original-price'}>&#36;{originalPrice}</span>
+        <br />
+        <span className={salePrice ? 'sale-price' : 'hide'}>SALE &#36;{salePrice}</span>
+        <br className={salePrice ? 'break' : 'hide'} />
```

Comment:
You could just combine this using one ternary operator like so:

` {salePrice
? (<span> ... (Sale price stuff)... <br />)
:  (<span> ... (Original price stuff) ... <br />)`